modules/proc-switching-bf2-nic.adoc

```diff
@@ -93,3 +93,5 @@ spec:
 <1> Optional: The PCI address of a specific card can optionally be specified, for example `ExecStart=/bin/bash -c '/etc/default/switch_in_sriov_config_daemon.sh nic 0000:5e:00.0 || echo done'`. By default, the first device is selected. If there is more than one device, you must specify which PCI address to be used. The PCI address must be the same on all nodes that are switching Bluefield-2 from DPU mode to NIC mode.
 
 . Wait for the worker nodes to restart. After restarting, the Bluefield-2 network device on the worker nodes is switched into NIC mode.
+
+. Optional: You might need to restart the host hardware because most recent Bluefield-2 firmware releases require a hardware restart to switch into NIC mode. 
```

Comment:
Hi @kquinn1204 @SalDaniele 

I think here we want to be more specific like reset the host with a cold boot via the out of band management interface

or something like that?